docs/hazmat/primitives/asymmetric/utils.rst

```diff
@@ -28,3 +28,50 @@ Asymmetric Utilities
     :param int s: The raw signature value ``s``.
 
     :return bytes: The encoded signature.
+
+.. function:: encode_ec_point(curve, x, y)
```

Comment:
If we wanted to tie this to EllipticCurvePublicNumbers, I think it would make sense to have something like an encode_point() or to_point() method on that which returned the encoded bytes, and a decode_point() or from_point() class method that took encoded bytes and a curve and constructed an instance from them.